entity-framework/core/querying/raw-sql.md

```diff
@@ -43,29 +46,29 @@ The following example passes a single parameter to a stored procedure. While thi
 var user = "johndoe";
 
 var blogs = context.Blogs
-    .FromSql("EXECUTE dbo.GetMostPopularBlogsForUser {0}", user)
+    .FromSqlInterpolated("EXECUTE dbo.GetMostPopularBlogsForUser {0}", user)
     .ToList();
 ```
 
-This is the same query but using string interpolation syntax, which is supported in EF Core 2.0 and above:
+This is the same query but using named interpolation syntax:
```

Comment:
I am not familiar with the term "named interpolation syntax". Can you provide some references? Otherwise, I suggest:
```suggestion
This is the same query but using string interpolation syntax and the `FromSqlInterpolated` method:
```